### ISSUE TITLE
suggestion: allow players to show a card they own.

### ISSUE THREAD

**Issue by admiralnlson** [zeel01/monarch#36]
Hi there,

I don't necessarily understand the rationale behind the showCard control being a GM-only option.

May I suggest changing the value of the 'showCard' disabled property from
`  !game.user.isGM`
to
`  !card.isOwner`
instead?

I suppose this could become a module setting also.

**Comment by zeel01** [comment]
So, the weird thing is that `card.isOwner` is always `true` for cards in your hand. Foundry kinda cheats its own permission system by making cards in a hand you own *owned copies* of the cards in the deck. So checking if a player owns a card is basically useless.

As for the "show card" thing... Yeah I could make it a setting whether or not to restrict this to the GM. The rationale is to prevent players from annoying each other by showing cards, as this will show everyone. But there are obviously situations where this would be useful, and it's probably not necessary to prevent players from doing it.

**Comment by admiralnlson** [comment]
> But there are obviously situations where this would be useful, and it's probably not necessary to prevent players from doing it.
Yes. I'd opt to leave this 'Prevent players from showing cards' option as off by default, with the rationale that it seems more natural to be permissive by default and leveraging the fact that 🦋 Monarch users probably haven't had enough time to get used to the default 'disabled for players' behavior since the feature is very new.
